Doc/tutorial/interpreter.rst

```diff
@@ -23,10 +23,10 @@ is an installation option, other places are possible; check with your local
 Python guru or system administrator.  (E.g., :file:`/usr/local/python` is a
 popular alternative location.)
 
-On Windows machines where you have installed Python from the :ref:`Microsoft Store
-<windows-store>`, the |python_x_dot_y_literal| command will be available. If you have
-the :ref:`py.exe launcher <launcher>` installed, you can use the :file:`py`
-command. See :ref:`setting-envvars` for other ways to launch Python.
+On Windows, if you installed Python from the :ref:Microsoft Store <windows-store>,
+the |python_x_dot_y_literal| command will be available. Alternatively, if you have the
```

Comment:
AFAIK, you might have both installed so it's not really "alternativelly". Also, we should keep the mention to setting environment variables.